### PR TITLE
Second stage of changes for the adoption of the content pack by Intel 471.

### PR DESCRIPTION
**Pull request by rhallick**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedIntel471/README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-Note: Support for this pack will be moving to the partner around 2021-04-23.
+Note: Support for this pack moved to the partner on 2021-05-05.  Please contact the partner directly via the support link on the right.
+
+# Intel 471
```
Comment (mchasepan) [inline Packs/FeedIntel471/README.md:3]: Please dont use H1 here @rhallick, H4 or 5 is best. 
And plz see the template I slacked you.

File: Packs/FeedIntel471/pack_metadata.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "Intel471 Feed",
-    "description": "This pack fetches actors indicators and malware-related indicators from intel471",
+    "description": "This pack fetches actor and malware related indicators from Intel 471.",
     "support": "xsoar",
```
Comment (moishce) [inline Packs/FeedIntel471/pack_metadata.json:4]:
```suggestion
    "support": "partner",
```

**Comment by rhallick** [comment]
So, should I now manually revert the version back to 1.1.4 and then run:
demisto-sdk update-release-notes -i Packs/FeedIntel471 -u revision
in order to increment the version back up to 1.1.5 and generate the release notes?

**Comment by rhallick** [comment]
OK, so getting:
================= Checking for missing release notes =================

Packs/FeedIntel471: [RN106] - Release notes were not found. Please run `demisto-sdk update-release-notes -i Packs/FeedIntel471 -u (major|minor|revision)` to generate release notes according to the new standard. You can refer to the documentation found here: https://xsoar.pan.dev/docs/integrations/changelog for more information.

Tried running:
demisto-sdk update-release-notes -i Packs/FeedIntel471 -u revision

But getting the following, even after making/committing additional small changes:

Starting to update release notes.

Either no changes were found in FeedIntel471 pack or the changes found should not be documented in the release notes file If relevant changes were made, please commit the changes and rerun the command

**Comment by rhallick** [comment]
Hi @moishce - do I need to do anything else now in order to complete the adoption of this content pack?

**Pull request merged**